### DISCUSSION TITLE
Any way to disable sorting json object field alphabetically?

### DISCUSSION THREAD

**Discussion by daxomy** [pocketbase/pocketbase#5492] [Q&A]
While saving an object to json field I need to prevent pocketbase sorting it alphabetically. Any suggestions?

**Comment by ganigeorgiev** [comment]
_See https://github.com/pocketbase/pocketbase/discussions/5403, https://github.com/pocketbase/pocketbase/discussions/4922#discussioncomment-9433526 and probably others._

It is not possible at the moment because this is the default Go `encoding/json` behavior.

[`encoding/json/v2`](https://github.com/golang/go/discussions/63397) may eventually change this but it is not an official proposal yet so whether the Go team will end up doing it is unknown.
